next.config.js

```diff
@@ -2,10 +2,9 @@
 const nextConfig = {
   output: 'export',
   reactStrictMode: true,
-  // for hosting under GitHub pages
-  basePath: '',
-  // Increase timeout for generating static pages from default 60s to avoid issues like:
-  // Restarted static page generation for /modules/xxx/y.y.y because it took more than 60 seconds
+  // 👇 Required for GitHub Pages under /bcr-ui
```

Comment:
Comment does not match the URL
```suggestion
  // 👇 Required for GitHub Pages under /bazel_registry_ui
```